mode/gfm/test.js

```diff
@@ -167,11 +137,6 @@
      "foo asfd:asdf bar");
 
   MT("notALink",
-     "[comment ```css]",
```

Comment:
permanently removed this test coz it's not a valid CommonMark codeBlock terminated... will need to update tokenizer later.